src/Framework/AspNetCoreAnalyzers/src/Analyzers/DiagnosticDescriptors.cs

```diff
@@ -233,4 +233,14 @@ internal static class DiagnosticDescriptors
         isEnabledByDefault: true,
         helpLinkUri: "https://aka.ms/aspnet/analyzers",
         customTags: WellKnownDiagnosticTags.Unnecessary);
+
+    internal static readonly DiagnosticDescriptor KestrelShouldListenOnIPv6AnyInsteadOfIpAny = new(
+        "ASP0028",
+        new LocalizableResourceString(nameof(Resources.Analyzer_KestrelShouldListenOnIPv6AnyInsteadOfIpAny_Title), Resources.ResourceManager, typeof(Resources)),
+        new LocalizableResourceString(nameof(Resources.Analyzer_KestrelShouldListenOnIPv6AnyInsteadOfIpAny_Message), Resources.ResourceManager, typeof(Resources)), // make a doc? https://aka.ms/aspnetcore-warnings/ASP0027
```

Comment:
Let's add ASP0028 to [list-of-diagnostics.md](https://github.com/dotnet/aspnetcore/blob/dc85807b334f1538ef9b5b2dde11b972e509d0d2/docs/list-of-diagnostics.md) as part of this PR. Then we can submit a PR to the [docs repo](https://github.com/dotnet/AspNetCore.Docs/tree/525450cac52374bb361d61d35f065d29ec3e7ed8/aspnetcore/diagnostics) and set up the aka.ms forwarding link as a follow up.

Reply:
I added ASP0025-ASP0028 to the doc. let me know if there were not added for a reason.

Reply:
here is the documentation PR: https://github.com/dotnet/AspNetCore.Docs/pull/34084